### PR TITLE
Add document download page

### PR DESCRIPTION
**Pull request by gov-cjwaszczuk**
## Summary 
Added landing page for document download links into admin.
## Screenshots 
image

### CODE REVIEW

File: app/templates/document_download_template.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	GOV.UK - Document download
+{% endblock %}
+
+{% block content %}
+	<div id="content">
+		<main role="main">
+      <div class="phase-banner">
+        <p>
+          <strong class="phase-tag">ALPHA</strong>
+          <span>This is a new service – your <a href="{{ url_for('main.support') }}">feedback</a> will help us to improve it.</span>
```
Comment (quis) [inline app/templates/document_download_template.html]: Lose the alpha banner. The services using document download might be in beta, they might be live; and ultimately users shouldn’t have to care.

File: app/templates/document_download_template.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{% extends "govuk_template.html" %}
+
+{% block head %}
+  <!--[if gt IE 8]><!-->
+  <link rel="stylesheet" media="screen" href="{{ asset_url('stylesheets/main.css') }}" />
```
Comment (quis) [inline app/templates/document_download_template.html]: Maybe not for this PR, but document download should use a separate CSS file – doesn’t need to load all the admin app stuff.

File: app/templates/document_download_template.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  <![endif]-->
+  <!--[if IE 8]>
+  <link rel="stylesheet" media="screen" href="{{ asset_url('stylesheets/main-ie8.css') }}" />
+  <![endif]-->
+  {% block meta %}
+  {% endblock %}
+{% endblock %}
+
+{% block page_title %}
+	GOV.UK - Document download
```
Comment (richardc0) [inline app/templates/document_download_template.html]: Check with Chris Heathcote on the Document download wording?

File: gulpfile.babel.js [hunk 4]
```diff
@@ -91,19 +92,26 @@ gulp.task('javascripts', () => gulp
...
-      paths.toolkit + 'stylesheets/'
-    ]
-  }))
-  .pipe(plugins.base64({baseDir: 'app'}))
-  .pipe(gulp.dest(paths.dist + 'stylesheets/'))
-);
+gulp.task('sass', () => {
+  const folders = [
+    '/stylesheets/notify/',
+    '/stylesheets/document-download/'
```
Comment (richardc0) [inline gulpfile.babel.js]: When we separate the app will we need both stylesheets? Do we want both?

**Pull request closed without merging**